### ISSUE TITLE
Support for Individualized Bayesian Knowledge Tracing

### ISSUE THREAD

**Issue by GokulNC** [CAHLR/pyBKT#12]
Thanks a lot for this work, it's really helpful.
Just wanted to know this.

Do you support student-specific learnable parameters as described in this paper:
[Individualized Bayesian Knowledge Tracing Models
](https://link.springer.com/chapter/10.1007/978-3-642-39112-5_18)
(Ofcourse, using EM algorithm only, need not be Gradient Descent)

**Comment by abadrinath947** [comment]
Yes, we do. Multilearn using student IDs is supported, but we will only be able to predict for the students for whom you have training data.

**Comment by zpardos** [comment]
To add to this.

Yes - we support multi learns based on student, but NO, Yudelson's approach to individualization is not supported at this time because it uses information across skills.
